src/domain/user/container/Login.js

```diff
@@ -23,29 +25,34 @@ class Login extends React.Component {
   }
 
   render() {
+    const prefix = config.env !== 'development' ? config.urlPrefix : '';
```

Comment:
@dwicao you can use this `const prefix = config.urlPrefix` only. And set on your `.env` `REACT_APP_URL_PREFIX=/kanca` so even on your local development, it still use `/kanca` prefix.